firmware/controllers/system/efi_gpio.cpp

```diff
@@ -577,7 +577,12 @@ uint8_t criticalErrorLedState;
 void initPrimaryPins(Logging *sharedLogger) {
 	logger = sharedLogger;
 #if EFI_PROD_CODE
-	enginePins.errorLedPin.initPin("led: CRITICAL status", LED_CRITICAL_ERROR_BRAIN_PIN, &(LED_ERROR_BRAIN_PIN_MODE));
+#ifdef EFI_LEDS_ACTIVE_LOW
+	static const pin_output_mode_e DEFAULT_OUTPUT_INVERTED = OM_INVERTED;
```

Comment:
lets's reduce the conditional compilation block or even completely remove it

let's just have ERROR_PIN_OUTPUT_MODE equal either OM_INVERTED or OM_DEFAULT and always use ERROR_PIN_OUTPUT_MODE?